llm/llama.cpp/generate.go

```diff
@@ -1,3 +1,6 @@
+//go:build !darwin
+// +build !darwin
```

Comment:
This should be omitted to build a CPU only serve

Reply:
Seems like metal support has been around for the lifetime of the apple silicon Macs, so I think this is ok.
https://support.apple.com/en-ca/HT205073